docker/requirements.txt

```diff
@@ -0,0 +1,11 @@
+numpy
```

Comment:
Can we pull in the requirement.txt file from repository root? It'd be nice to only have one place to document/manage build dependencies.

Reply:
Change the build context so that it can see those files